web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.tsx

```diff
@@ -44,6 +48,17 @@ export const ConnectMyComputerContextProvider: FC<{
   rootClusterUri: RootClusterUri;
 }> = props => {
   const { mainProcessClient, connectMyComputerService } = useAppContext();
+  const [isSetupDoneAttempt, checkIfSetupIsDone, setSetupDoneAttempt] =
```

Comment:
How about `isAgentConfiguredAttempt`? `isSetupDone` makes it seem like setup was in progress at some point and we're polling to see if has finished.

Reply:
Done